example/src_py/ctypes_extended.py

```
@@ -58,6 +59,9 @@ def __get__(self, instance, owner):
         return value
 
     def __set__(self, instance, value):
+        if value not in self.range:
+            raise ValueError('value to set is outside of range')
+
```

in example/src_py/

make a test_extended.py file.
update ectypes_gps.py on how to use range attribute.
if i have range 0 to 10 then it should throw an error of any number outside of that range.


```

import ectypes_gps.py as gps

import some_sort_of_testing_framework?

 

health = gps.Health_M()

health.dac = 11.9

try:

  health.dac = 10

except:

  expected error

```

will range allow numbers inside of